Tests/WebSocketKitTests/AsyncWebSocketKitTests.swift

```diff
@@ -91,9 +92,9 @@ final class AsyncWebSocketKitTests: XCTestCase {
             return XCTFail("couldn't get port from \(String(reflecting: server.localAddress))")
         }
         try await WebSocket.connect(to: "ws://localhost:\(port)", on: self.elg) { (ws) async in
-            ws.onPong {
+            ws.onPong {socket, _ in
                 do {
-                    try await $0.close()
+                    try await socket.close()
```

Comment:
Why?

Reply:
Dunno never touched any of the ping pong logic that I recall.